### ISSUE TITLE
Permission required

### ISSUE THREAD

**Issue by Tangsir-RS** [annavaughan/CH4Net#1]
![image](https://github.com/annavaughan/CH4Net/assets/25479042/3ddbe642-cc13-43f7-96e6-56779398e7ea)
We do not have sufficient permissions to view this page.

**Comment by chloeahampton** [comment]
yes, I am noticing the same thing and am very interested in accessing this data!

**Comment by GunnJack** [comment]
I am also interested in accessing the data. Thank you!

**Comment by J6767** [comment]
Same issue.

**Comment by lmannyl** [comment]
Same problem not fixed yet. 13, Dec, 2023

**Comment by marcoruizrueda-ist** [comment]
Hi guys, the dataset was uploaded by the authors via HuggingFace: https://doi.org/10.57967/hf/2117
Please let's keep in contact to discuss how to load the label npy files. In my case, I just uploaded the npy labels via numpy but only had zero values. Any else with the same issue?

**Comment by oguzhannysr** [comment]
@marcoruizrueda-ist ,if you want, try installing it with zenodo as well.

**Comment by oguzhannysr** [comment]
![image](https://github.com/annavaughan/CH4Net/assets/53774968/7f8860ce-b14c-4483-94b2-18a6b7e1a0af)
@marcoruizrueda-ist , I downloaded the data you uploaded to huggingface and wanted to start training on colab. But I'm getting an error. I saw roads like Turkmenistan on train.py, but I don't have them either. What do I need to correct in that code?

**Comment by marcoruizrueda-ist** [comment]
I didn't upload that data, it was the authors. That hugging face is being maintained by them. In fact, I'm having problems loading the labels since all the values of the bumpy arrays seem to be only zero values.

**Comment by Renat2001** [comment]
Are you all also getting different metrics on test and val data? It seems like the dataset (https://doi.org/10.57967/hf/2117) is a bit different than the one described in the paper.

**Comment by aslazarte21** [comment]
The original files by the researchers have a separate folder for positive and negative images which are not included in the uploaded files. Do you have a workaround for this?
